pkg/metrics/metrics.go

```diff
@@ -938,6 +942,14 @@ func NewLegacyMetrics() *LegacyMetrics {
 			Help:      "Number of total datapath update timeouts due to FQDN IP updates",
 		}),
 
+		ConntrackMapCapacity: metric.NewGaugeVec(metric.GaugeOpts{
```

Comment:
I wonder if we could use [this existing metrics](https://github.com/cilium/cilium/blob/main/pkg/metrics/metrics.go#L1222-L1228) for this, instead of creating a new one?